spec/models/book_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe Book do
+
+  describe 'validations' do
+
+    it 'pass with a title' do
+      book = Book.new(title: 'title')
+      expect(book).to be_valid
+    end
+
+    it 'fail without a title' do
+      book = Book.new
+      expect(book).to be_invalid
+    end
```

Comment:
Like these tests. Also like that you aren't saving to the DB and realize that you can check validity on an instance of Book that hasn't yet been persisted.